packages/namekit-react/package.json

```diff
@@ -41,8 +41,10 @@
   "dependencies": {
     "@headlessui-float/react": "0.11.4",
     "@headlessui/react": "1.7.17",
+    "@namehash/ens-webfont": "workspace:*",
     "@namehash/ens-utils": "workspace:*",
-    "@namehash/ens-webfont": "workspace:*"
+    "react": "^18.2.0",
```

Comment:
@FrancoAguzzi Are you 100% sure you added `react` and `react-dom` as dependencies correctly? Shouldn't they have been added as `devDependencies` rather than `dependencies`?

I already see them listed as `devDependencies`.

What's happening here? PLEASE REVIEW YOUR OWN WORK!